rt/transports/http-hc/src/main/java/org/apache/cxf/transport/http/asyncclient/AsyncHTTPConduit.java

```diff
@@ -245,18 +244,6 @@ public boolean isRepeatable() {
         message.put(CXFHttpRequest.class, e);
     }
 
-    private void propagateJaxwsSpecTimeoutSettings(Message message, HTTPClientPolicy csPolicy) {
```

Comment:
The `csPolicy` is shared within same HTTP Async Conduit instance, however the timeouts are per message and override each other (in case each message has contextual timeouts set).